src/stcal/saturation/saturation.py

```diff
@@ -83,64 +83,85 @@ def flag_saturated_pixels(
 
     for ints in range(nints):
         # Work forward through the groups for initial pass at saturation
+
+        # We want to flag saturation in all subsequent groups after
+        # the one in which it was found.  Use this boolean array to
+        # keep a running tally of pixels that have saturated.
+        previously_saturated = np.zeros(data[0, 0].shape, dtype='bool')
```

Comment:
I think it's clearer to use `shape=(row,col)`, rather than `shape=data[0,0].shape`.

Reply:
Sounds good; I changed it.